syntax = "proto3";

package NSound;

import "protos/service/stream.proto"; 
import "protos/service/context.proto"; 

message TServiceMessage {

    oneof Request {
        NService.TStreamMessage stream_message = 1;
        NService.TContextMessage context_message = 2;
    }
}